node/lib/util/git_util.js

```diff
@@ -911,3 +911,66 @@ exports.configIsTrue = co.wrap(function*(repo, configVar) {
     return (configured === "true" || configured === "yes" ||
             configured === "on");
 });
+
+/**
+ * Fetch the remote having the specified `repo`.
```

Comment:
"Fetch and return a list of remote refs in the origin for the specified `repo` "

Reply:
also maybe rename this to getRemoteRefs?